### PR TITLE
backend - get rooms available of hotel is determined (check condition: "start date, end date, person quantity per room, room quantity

### PR DESCRIPTION
**Pull request by vietvu1996**
Please review my code

### CODE REVIEW

**Review by dinhhieu240901 — APPROVED** [review]
nice

**Review by thaonguyen011 — APPROVED** [review]
good

File: src/main/java/com/cgtravelokaservice/service/implement/RoomBookingService.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        this.roomContractService = roomContractService;
+        this.hotelRepo = hotelRepo;
+    }
+
+
+    public RoomBookingResponse displayListRoom(RoomBookingRequestDTO bookingRequest) {
+        List<Room> availableRooms = new ArrayList<>();
+        List<Room> rooms = roomRepo.findByHotelId(bookingRequest.getHotelId());
+
+        for (Room room : rooms) {
```
Comment (LamChiHung) [inline src/main/java/com/cgtravelokaservice/service/implement/RoomBookingService.java:34]: This code is wrong logic, 1 room in the list not available is not mean all rooms unavailable

**Review by LamChiHung — COMMENTED** [review]
Have some wrong logic, please check again

File: src/main/java/com/cgtravelokaservice/service/implement/RoomBookingService.java (hunk 1 above)
Comment (vietvu1996) [reply src/main/java/com/cgtravelokaservice/service/implement/RoomBookingService.java:34]: Got it

**Review by LamChiHung — APPROVED** [review]
well done

**Pull request merged**